daprdocs/content/en/getting-started/install-dapr-selfhost.md

```diff
@@ -137,7 +137,11 @@ ls $HOME/.dapr
 {{% codetab %}}
 
```

Comment:
Powershell

Reply:
Suggest being explicit with the commands

Reply:
Done!